### DISCUSSION TITLE
Select2 multiple not showing data name on edit

### DISCUSSION THREAD

**Discussion by rickyviz** [Laravel-Backpack/community-forum#279] [Q&A (Help)] [answered]
I make select2 multiple tag,and save the tag on diffrent table 1-N relation
![image](https://user-images.githubusercontent.com/89818695/138378375-e2b816c4-bfec-4f4c-b602-a45a275b9eb2.png)
![image](https://user-images.githubusercontent.com/89818695/138378404-ec93add1-ae76-4bc9-9fd4-01ae523a8f66.png)

**Comment by pxpm** [comment, marked as answer]
@rickyviz it migth be your returned results from api.

I would suggest you setup the api using the `FetchOperation` https://backpackforlaravel.com/docs/4.1/crud-operation-fetch#how-to-use-1
